layers/+misc/multiple-cursors/packages.el

```diff
@@ -34,3 +34,25 @@
           (define-key state-map (car keybinding) (cdr keybinding))))
       (add-hook 'prog-mode-hook 'turn-on-evil-mc-mode)
       (add-hook 'text-mode-hook 'turn-on-evil-mc-mode))))
+
+(defun multiple-cursors/init-multiple-cursors ()
+  (use-package multiple-cursors
+    :defer t
+    :init
+    (progn
+      (spacemacs/set-leader-keys "grr" 'mc/mark-more-like-this-extended)
```

Comment:
This binds the keys to `SPC g r r` (but they are probably overwritten by the `git` layer, because it's still bound to the expected command: `spacemacs/smerge-transient-state/body`

`evil-mc` seems to bind the keys to an `evil-mc-key-map` variable:
https://github.com/gabesoft/evil-mc/blob/master/evil-mc.el#L79

Reply:
This is a problem. They are overwritten by `source-control` layer. Actually my advice is to bind `mc` under `SPC x m`. `SPC g` is mostly used by source control layers. However if you say that they should be under `SPC g r` (as evil-mc resides here), I will update the bindings.

Reply:
The `evil-mc` bindings are under `gr` without the `SPC` prefix. But since they are evil editing style specific they aren't an option for emacs editing style users.

They should not be under `SPC g r`, since as you say `SPC g` is for `git/versions-control` keys.

Sorry for not being clear, I meant that:
` (spacemacs/set-leader-keys "grr" 'mc/mark-more-like-this-extended)`
binds them to `SPC g r r`, which we shouldn't do.

`iedit` is bound to `SPC s e` and `SPC s m` seems to be available, would be better to try and group them together. Or is there a benefit to have them under `SPC x m`?

Reply:
The thing is `SPC x` is about text manipulation and so does `mc`. `SPC s` is the searching prefix. However since `iedit` is under `SPC s`, this makes `SPC m` a good candidate.

Reply:
`SPC s` is called `search/symbol`, so maybe `iedit` is there because multiple cursors usually change multiple functions/symbols/variables/etc.

Where do you see that `SPC x m` is called `move`?

Reply:
It seems is no longer a prefix. It used to be. https://github.com/syl20bnr/spacemacs/blob/c7a103a772d808101d7635ec10f292ab9202d9ee/layers/%2Bdistributions/spacemacs-base/keybindings.el#L64

Reply:
It seems to have been removed from the develop branch.
https://github.com/syl20bnr/spacemacs/blob/b4907d0540bae38588ed50fad063883c76c41867/layers/%2Bspacemacs/spacemacs-defaults/keybindings.el#L81-L88

Reply:
I've changed the bindings for now. Someone may want to test it.